rebar.config

```diff
@@ -12,6 +12,7 @@
 ]}.
 
 {deps, [
+    {multihash, {git, "https://github.com/helium/erlang-multihash.git", {tag, "v2.0.2"}}},
```

Comment:
this is actually on hex.pm so use the simpler declaration

Reply:
fixed